kosat-web/.yarnrc

```diff
@@ -0,0 +1 @@
+network-timeout 500000
```

Comment:
Perhaps 8 minutes is a bit too much?

Reply:
The default timeout is 30s. The GHA runners might "fall asleep" during the execution, so we need to increase the timeout. It does not really matter to which amount, since this operation is probably will always be successful on GHA. MUI users report the following solution: https://github.com/mui/material-ui/issues/12432#issuecomment-411046157

Actually, this .yarnrc file with timeout setting should only exist on GHA runner, so probably it would be better the create such file directly in a separate GHA job step, without including it into our repository. Especially note that this is only needed on Windows runners. Maybe I will try and fix it later (without a PR).